RELEASE.md

```diff
@@ -0,0 +1,11 @@
+# Releasing
+
+1. Bump version code in `publish.gradle`
+1. Update examples in `README.md` to use the latest version
+1. Create an entry in `CHANGELOG.md` containing the changes in that release. This entry needs to follow a certain pattern which can be found in `publish.gradle`.
```

Comment:
Maybe worth mentioning that you can verify the last changelog entry using `printChangelog` task?

Reply:
Addressed in 982ee6b